bin/validate-json

```diff
@@ -221,7 +221,9 @@ try {
     $validator->check($data, $schema);
 
     if ($validator->isValid()) {
-        echo "OK. The supplied JSON validates against the schema.\n";
+        if(isset($arOptions['--verbose'])) {
+            echo "OK. The supplied JSON validates against the schema.\n";
+        }
     } else {
         echo "JSON does not validate. Violations:\n";
         foreach ($validator->getErrors() as $error) {
```

Comment:
Looking great :-). 

While you're at it, can we also have a `--quiet` option that suppresses the error output?